### PR TITLE
Implement Composer dependency management (#376)

### PR DESCRIPTION
**Pull request by greatislander**

### CODE REVIEW

**Comment by greatislander** [comment]
@bdolor @hughmcguire Please test when you can. I'm pretty confident in these changes but the more eyes the better! You'll need to run `composer install` in the plugin directory once you've cloned this branch or PR. Thanks.

**Comment by connerbw** [comment]
Hey, awesome!

As as aside I read this today.

http://mouf-php.com/optimizing-composer-autoloader-performance

Don't forget to add --optimize to the deploy process?

Cheers.

**Comment by connerbw** [comment]
And of course this:

https://blog.engineyard.com/2014/composer-its-all-about-the-lock-file

Buy you've already included the lock file so kudos.

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/pressbooks/htmlawed"
+    },
+    {
+      "type": "vcs",
+      "url": "https://github.com/pressbooks/PrinceXMLPhp"
+    }
+  ],
```
Comment (connerbw) [inline composer.json]: [Good use of forking](http://snippets.khromov.se/composer-use-your-own-fork-for-a-package/). .

**Comment by connerbw** [comment]
Reviewed.

**Comment by bdolor** [comment]
I'm late to the party...I'll blame it on the timezone difference. Can you confirm that exports are working for you after this PR?

**Comment by greatislander** [comment]
They are for me and for @connerbw. Are you running into any issues, @bdolor?

**Comment by bdolor** [comment]
Prince is failing for me, though I'm only seeing a PHP Warning — other exports look ok. Prince produces no export file. 
screen shot 2016-04-05 at 11 20 28 am

**Comment by greatislander** [comment]
@bdolor That [line number](https://github.com/pressbooks/pressbooks/blob/dev/includes/modules/export/class-pb-export.php#L718) is related to book language. Do you have a language set for the book your testing with?

**Comment by bdolor** [comment]
My inability to produce a PDF with this update is related to the version of Prince. This update requires Prince version greater than 9.0 rev 5. Further testing with different Prince versions would reveal which version of Prince is the absolute minimum version required in order for it to work, but upgrading Prince to 10r7 on my development server resolved this for me. Will have to ensure to upgrade PrinceXML on production servers before pulling in this change. 

Reviewed :+1:

**Pull request merged**